tidb-lightning.toml

```diff
@@ -138,10 +138,11 @@ checksum-table-concurrency = 16
 # if set true, checksum will do ADMIN CHECKSUM TABLE <table> for each table.
 checksum = true
 # if set to true, compact will do level 1 compaction to tikv data.
-# if this setting is missing, the default value is also true.
+# if this setting is missing, the default value is also false.
```

Comment:
Please remove the "also"

Reply:
Removed in https://github.com/pingcap/tidb-lightning/pull/202/commits/d7b952af22deacdd093c01a497f77fa03c146edb